instance/models/load_balancer.py

```diff
@@ -91,17 +99,46 @@ class LoadBalancingServer(ValidateModelMixin, TimeStampedModel):
     objects = LoadBalancingServerManager()
 
     domain = models.CharField(max_length=100, unique=True)
-    # The username used to ssh into the server
-    ssh_username = models.CharField(max_length=32)
-    # Whether new backends can be assigned to this load-balancing server
-    accepts_new_backends = models.BooleanField(default=True)
-    # A random postfix appended to the haproxy configuration file names to avoid clashes between
-    # multiple instance managers (or multiple concurrently running integration tests) sharing the
-    # same load balancer.
+
+    ssh_username = models.CharField(
+        max_length=32,
+        help_text='The username used to SSH into the server.'
+    )
+
+    accepts_new_backends = models.BooleanField(
+        default=True,
+        help_text='Whether new backends can be assigned to this load-balancing server.'
+    )
+
     fragment_name_postfix = models.CharField(
         max_length=8,
         blank=True,
         default=functools.partial(generate_fragment_name, length=8),
+        help_text=(
+            'A random postfix appended to the haproxy configuration file names to avoid clashes between '
+            'multiple instance managers (or multiple concurrently running integration tests) sharing the '
+            'same load balancer.'
+        )
+    )
+
+    configuration_version = models.PositiveIntegerField(
+        default=1,
+        help_text=(
+            'The current version of configuration for this load balancer. '
+            'By default, a version of 0 means that the load balancer is clean, '
+            'i.e. it does not currently require a reconfiguration. '
+            'The version value is essentially equivalent to the number of requests made '
+            'to reconfigure the load balancer since it last was in progress.'
```

Comment:
This docstring seems out of date.

It says "By default, a version of 0 ", but the version starts at 1 by default and never gets reset to zero.

It also says "The version value is essentially equivalent to the number of requests made to reconfigure the load balancer since it last was in progress." but that doesn't seem to be the case anymore, since the latest code seems to only ever increment the version number. So it would be more accurate to say "The version value is the total number of requests ever made to reconfigure the load balancer".

Reply:
Yeah, it's definitely out of date, sorry! I had tried a different technique before this one but forgot to update the help text.